purchase_receipt_expectation_manual/wizards/purchase_order_manual_receipt.py

```diff
@@ -0,0 +1,234 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from collections import defaultdict
+
+from odoo import _, fields, models
+from odoo.tools.float_utils import float_compare
+
+
+class PurchaseOrderManualReceipt(models.TransientModel):
+    _name = "purchase.order.manual.receipt.wizard"
+    _description = "PO Manual Receipt Wizard"
+
+    line_ids = fields.One2many(
+        "purchase.order.manual.receipt.wizard.line",
+        "wizard_id",
+    )
+
+    auto_confirm_picking = fields.Boolean(
+        default=True, help="Automatically confirms the picking after creation."
+    )
+
+    checks_result = fields.Selection(
+        [("success", "Success"), ("failure", "Failure")],
+        default="success",
+    )
+
+    checks_result_msg = fields.Text(
+        help="Stores error messages result from executed checks"
+    )
+
+    purchase_order_id = fields.Many2one(
+        "purchase.order",
+        required=True,
+    )
+
+    scheduled_date = fields.Datetime(
+        default=fields.Datetime.now,
+        required=True,
+    )
+
+    #######################
+    # BUTTONS AND ACTIONS #
+    #######################
+
+    def open_form_view(self):
+        self.ensure_one()
+        return {
+            "name": _("Manual Receipt"),
+            "type": "ir.actions.act_window",
+            "res_id": self.id,
+            "res_model": "purchase.order.manual.receipt.wizard",
+            "target": "new",
+            "view_mode": "form",
+        }
+
+    def open_picking_form_view(self, picking):
+        self.ensure_one()
+        return {
+            "name": _("Picking"),
+            "type": "ir.actions.act_window",
+            "res_id": picking.id,
+            "res_model": "stock.picking",
+            "view_mode": "form",
+        }
+
+    def button_check(self):
+        """Runs checks on the wizard data, then opens the wizard again"""
+        self.ensure_one()
+        errs = self._execute_all_checks()
+        self.write(self._prepare_post_check_vals(errs))
+        return self.open_form_view()
+
+    def button_confirm(self):
+        """Confirms wizard data and creates new picking"""
+        self.ensure_one()
+        # Setting `checks_result` to "success" because the user is allowed to
+        # ignore warnings; then, we'll run pre-confirm checks and, if any of
+        # them fails, field `checks_result` should be changed
+        self.checks_result = "success"
+        errs = self._execute_pre_confirm_checks()
+        self.write(self._prepare_post_check_vals(errs))
+        if self.checks_result != "success":
+            # Some pre-confirm check failed: reopen the wizard
+            self.checks_result_msg += _("\n\nThese checks cannot be skipped!")
```

Comment:
```suggestion
            self.checks_result_msg += "\n\n" + _("These checks cannot be skipped!")
```
It's better for translations